CommunicationLayer/CommDeviceControl/RadioCommDevice.cpp

```diff
@@ -0,0 +1,55 @@
+#include <QDebug>
+#include <QTimer>
+#include "RadioCommDevice.h"
+
+namespace
+{
+   const QString DEFAULT_PORT_NAME = "/dev/pts/7";
```

Comment:
```c++
#ifdef TESTING
   const QString DEFAULT_PORT_NAME = "/dev/pts/7";
#else
   const QString DEFAULT_PORT_NAME = "dev/ttyS0"; // Or whatever the one is on the RaspPi
#endif
```

One option is to do this so the actual port that is used in production is used by default. Only if TESTING is defined then it would using the fake comm port. 

TESTING would probably be passed into qmake or in a qtcreator option.